homeassistant/components/zwave_js/entity.py

```diff
@@ -152,6 +152,12 @@ def available(self) -> bool:
             and self.info.node.status != NodeStatus.DEAD
         )
 
+    @property
+    def extra_state_attributes(self) -> Optional[Dict[str, str]]:
+	"""Return additional entity attributes"""
+    return {"node_id": self.info.node.node_id}
```

Comment:
What's the use case for adding node id to state attributes. We've discussed this a coupleof times already and always concluded that it's not needed.

Reply:
```suggestion
        return {"node_id": self.info.node.node_id}
```